modules/installation/proc-rhdh-deploy-aks-operator.adoc

```diff
@@ -4,167 +4,47 @@
 [id='proc-rhdh-deploy-aks-operator_{context}']
 = Deploying {product-short} on {aks-short} with the Operator
 
-You can deploy your {product-short} on {aks-short} using the {product} Operator.
+The {product} Operator installation requires the Operator Lifecycle Manager (OLM) framework.
```

Comment:
Consider moving to the Prerequisites section.

```suggestion
```

Reply:
Fixed: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/729#discussion_r1867617883